### ISSUE TITLE
TypeError during Building gene structure plot

### ISSUE THREAD

**Issue by james-cranley** [kharchenkolab/Baysor#62]
New user. Running `baysor preview`. I get this error which seems to be because the `plot_expression_vectors ` function expects `gene_names` to be a vector of `InlineStrings.String7`, but it is receiving a vector of `String`.

Running Julia 1.8.5 and having installed baysor via Pkg.

The `molecules` file I have passed is in the correct format based on the examples in your github examples folder.

What can I do to fix this please?

```
[17:15:02] Info: Building gene structure plot
[17:15:09] Error: TypeError(Symbol("keyword argument"), :gene_names, Vector{String}, InlineStrings.String7["ACKR1", "ADGRA3", "ADGRD1", "ADGRL1", "ADGRL3", "ADRB1", "AGT", "AGTR1", "BNC1", "C1QA", "CA4", "CACNA1D", "CALCA", "CALCB", "CALCRL", "CCL2", "CCL21", "CCR2", "CD8A", "CDH5", "CGNL1", "CHAT", "CHRM2", "CLDN5", "CX3CL1", "FABP4", "FAP", "FASN", "FGF7", "GALR1", "GCG", "GJA5", "GLP1R", "GLS", "GLUL", "GRIA3", "GRIK2", "HCN1", "HCN4", "ICAM1", "IFIT1", "IFIT3", "INHBA", "ISG15", "ISG20", "ITLN1", "KCNJ3", "KCNJ8", "MEOX2", "MYH11", "NEBL", "NEFM", "NGF", "NGFR", "NLGN1", "NRXN1", "NRXN3", "NTRK1", "PCDH15", "PDGFRA", "PDZRN4", "PLA2G5", "PLVAP", "POSTN", "PRPH", "RERGL", "RGCC", "RGS5", "SDC1", "SELE", "SEMA3G", "SLC10A4", "SLC18A1", "SLC18A2", "SLC18A3", "SLC1A3", "SLC38A2", "SLC5A7", "SLC6A4", "SMOC1", "SNAP25", "SST", "SSTR2", "STEAP4", "SULF1", "TACR2", "TBX3", "TCF15", "TFF3", "TH", "TIMD4", "TTN", "VCAM1", "VIP", "VIPR2", "VWF"])
|                 
|                 (::Baysor.var"#plot_expression_vectors##kw")(::NamedTuple{(:gene_names, :labels, :ylabel, :min_expr_frac, :alpha), Tuple{Vector{InlineStrings.String7}, Vector{String}, String, Float64, Float64}}, ::typeof(Baysor.plot_expression_vectors), ::Vector{Int64}, ::Vararg{Vector{Int64}}) at plots.jl:121
|                 plot_num_transcript_overview(genes::Vector{Int64}, confidences::Vector{Float64}, gene_names::Vector{InlineStrings.String7}; alpha::Float64) at diagnostic_plots.jl:23
|                 plot_num_transcript_overview(genes::Vector{Int64}, confidences::Vector{Float64}, gene_names::Vector{InlineStrings.String7}) at diagnostic_plots.jl:21
|                 run_cli_preview(args::Vector{String}) at preview.jl:117
|                 run_cli(args::Vector{String}) at common.jl:156
|                 run_cli() at common.jl:137
|                 top-level scope at baysor:3
|                 eval at boot.jl:368 [inlined]
|                 include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String) at loading.jl:1428
|                 _include(mapexpr::Function, mod::Module, _path::String) at loading.jl:1488
|                 include(mod::Module, _path::String) at Base.jl:419
|                 exec_options(opts::Base.JLOptions) at client.jl:303
|                 _start() at client.jl:522
└ Baysor /lustre/scratch126/cellgen/team205/jc48/jupyter/baysor/julia-1.8.5/packages/Baysor/K7Vqu/src/cli/common.jl:170

```

**Comment by ccbvib** [comment]
I am also experiencing the same error that @james-cranley is reporting. I am also using Julia 1.8.5 and the error occurs even working with the `mRNA_coords_raw_counting.csv` FISH test data and FISH config file.

**Comment by VPetukhov** [comment]
@james-cranley and @ccbvib , this was an error caused by an unexpected update in the CSV reading package. It should be fixed in [v0.6.0](https://github.com/kharchenkolab/Baysor/releases/tag/v0.6.0), could you please try that?
